library/src/scripts/styles/buttonStyles.ts

```diff
@@ -39,34 +39,36 @@ export function buttonStyles(theme?: object) {
     return { padding, sizing, border };
 }
 
+type Unit = string | number;
+
 export interface IButtonType {
     fg: string;
     bg: string;
-    spinner: ColorHelper;
+    spinner: string;
     border: {
         color: string;
-        width: string;
+        width: Unit;
         style: string;
-        radius: string;
+        radius: Unit;
     };
     hover: {
-        fg: string;
-        bg: string;
+        color: string;
```

Comment:
Why change this? fg can bd an icon color, or a border, I like it better the way it is.